API/app/Http/Controllers/Api/ShoppingListController.php

```diff
@@ -123,9 +127,10 @@ public function store(Request $request)
     {
         Log::info("Received shopping-lists request from IP address " . $request->ip());
 
-        $userId = $request->user_id;
+        // Grab user from sanctum
+        $user = Auth::user();
 
-        Log::info("User ID retrieved from JWT: " . print_r($userId, true));
+        Log::info("User ID retrieved from JWT: " . print_r($user->user_id, true));
```

Comment:
Change this logging to only show on DEBUG mode (add constant to this file if necessary).